[debops.apt_proxy] Drop unneeded dependency `python3-future`

ypid opened the pull request:
Closes #181

AnBuKu commented:
Hmmm... Sure, somehow boring on one hand and on the other hand rewarding in terms of "Hey, one role more!"

Or, we do it the same way as DebOps Status page...?

Looking at https://debops.org/status.html I see "There are 118 roles in the DebOps project." The 118 counted by HTML code `<span id="role_count">118</span>` or rather the embedded script (see part of script below), respectively https://api.debops.org/v0/roles/count:debops, automation might work for GitHub and https://pypi.python.org/pypi/debops as well. - Or does `lots` all that?

```
<script>
(function() {
  }
  function update_role_count() {
    var xhr = new XMLHttpRequest();
    xhr.open('get', 'https://api.debops.org/v0/roles/count:debops', true);
    xhr.onload = function() {
      var status = xhr.status;
      if (status == 200) {
        document.getElementById('role_count').innerHTML = xhr.response;
      }
    };
    xhr.send();
  }
</script>

```

drybjed commented:
I'm not sure if GitHub or PyPI will allow embedded JavaScript in their respective pages.

AnBuKu commented:
How could we find out? ;-) `</ teasing>`

ypid committed:
Fix typo

ypid committed:
Drop unneeded dependency `python3-future`. Closes #181.

```Shell
$ python3
Python 3.4.2 (default, Oct  8 2014, 10:45:20)
[GCC 4.9.1] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import __future__
>>> __future__.__file__
'/usr/lib/python3.4/__future__.py'
$ dpkg -S /usr/lib/python3.4/__future__.py
libpython3.4-minimal:amd64: /usr/lib/python3.4/__future__.py
```

drybjed reviewed (approved):
Thanks for checking it!
